core/unit_test/hip/TestHIP_UnifiedMemory_ZeroMemset.cpp

```diff
@@ -30,7 +30,10 @@ TEST(hip, unified_memory_zero_memset) {
 
   constexpr size_t N = static_cast<size_t>(1024 * 1024);  // size doesn't matter
   std::vector<int> v(N, 1);  // initialize to non-zero
-  Kokkos::View<int*, Kokkos::HIPSpace> a(v.data(), N);
+
+  // FIXME_HIP: this should also work with HIPSpace once SpaceAccessibility says
+  // it's okay.
```

Comment:
SpaceAccessibilty doesn't impose any restrictions here (yet). We only start checking memory spaces in #7480 and with #7662 this would remain valid anyway.
My concern is rather that using `HIPSpace` here would make the test fail with `HSA_XNACK=0` if we only warn and not abort.

Reply:
In other words, there is no way to use `SpaceAccessibility` to detect such problems.

Reply:
yeah, you are right. But independent of the space we use here, the test will fail if we are running without `XNACK`.
But we want to know if it behaves correctly with `XNACK` enabled so the test still provides coverage

Reply:
I removed this comment and added a `HIPSpace` test to track it for posterity.